app.py

```diff
@@ -60,10 +61,9 @@ def inner():
 			for line in iter(proc.stdout.readline,''):
 				time.sleep(1)  # Don't need this just shows the text streaming
 				yield line.rstrip() + '<br/>\n'
-
 	else:
 		return redirect(url_for('index'))
-	return Response(inner(), mimetype='text/html')  # text/html is required for most browsers to show th$
+	return Response(inner())  # text/html is required for most browsers to show th$
 
```

Comment:
According to me, if it is set by default, them i felt that there is no need for an extra prameter to pass. What are your views @abishekvashok